components/canvas/Cargo.toml

```diff
@@ -32,7 +32,7 @@ lyon_geom = "1.0.4"
 num-traits = { workspace = true }
 pathfinder_geometry = "0.5"
 pixels = { path = "../pixels" }
-raqote = "0.8.2"
+raqote = { version = "0.8.3", git = "https://github.com/jrmuizel/raqote.git", rev = "a1e754adbb4a6808b3f5a74d33ac37b32843ed8e" }
```

Comment:
What's the story on how close this is to an actual release?

Reply:
raqote's CI pipeline is currently failing because of https://github.com/servo/pathfinder/pull/548 being unreleased. Perhaps we should release a new version of pathfinder first?

Reply:
raqote 0.8.4 has released to crates.io